aaq/tasks.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from celery.exceptions import SoftTimeLimitExceeded
+from django.conf import settings
+from requests.exceptions import RequestException
+from temba_client.exceptions import TembaHttpError
+
+from ndoh_hub.celery import app
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded, TembaHttpError),
```

Comment:
`TembaHttpError` is a Rapidpro API client error, shouldn't be possible here

Reply:
Removed